frontend/src/components/application/ListBox.tsx

```diff
@@ -34,7 +34,7 @@ const ListBox: FunctionComponent<Props> = ({ appId, items }) => {
             }
             return (
               <div
-                className={`grid w-full grid-cols-[36px_calc(100%_-_36px_-_36px)_36px] bg-flathub-white p-4 shadow-md first:rounded-tl-xl first:rounded-tr-xl last:rounded-bl-xl last:rounded-br-xl dark:bg-flathub-arsenic ${
+                className={`grid h-full w-full grid-cols-[36px_calc(100%_-_36px_-_36px)_36px] bg-flathub-white p-4 shadow-md first:rounded-tl-xl first:rounded-tr-xl last:rounded-bl-xl last:rounded-br-xl dark:bg-flathub-arsenic ${
```

Comment:
I think this is an improvement in some places, but only really works with an additional `items-center`

Without
![image](https://user-images.githubusercontent.com/5943908/235359905-1c91a27f-50d1-4ea9-915d-75e3295fd0da.png)

with
![image](https://user-images.githubusercontent.com/5943908/235359922-ad8a1f75-1341-46c6-9580-a0b8d5aef84a.png)